srcpkgs/ncdc/template

```diff
@@ -0,0 +1,18 @@
+# Template file for 'ncdc'
+pkgname=ncdc
+version=1.20
+revision=1
+build_style=configure
+configure_args="--prefix=/usr --with-geoip"
```

Comment:
prefix would be redundant in that case because it is already set by build-style `gnu-configure`.